purchases/src/main/kotlin/com/revenuecat/purchases/common/events/EventsManager.kt

```diff
@@ -27,6 +30,7 @@ import kotlinx.serialization.modules.polymorphic
  * @property postEvents Function for sending events to the backend.
  */
 internal class EventsManager(
+    private val appSessionID: UUID = UUID.randomUUID(),
```

Comment:
This may potentially be recreated if someone reconfigures the SDK. It's rare, but possible... Maybe it's ok to do it like this anyway since that's a pretty rare case. The alternative being storing this into a static variable.